cypress/component/home/Home.cy.tsx

```diff
@@ -8,37 +8,188 @@ import {
   updatedUserFeatures
 } from "../../../redux/slices/userSlice";
 import history from "../../../components/navigation/history";
-import { mockUserFeatures1 } from "../../../mock-data/trainee-profile";
+import {
+  mockUserFeaturesLtftPilot,
+  mockUserFeaturesNonSpecialty,
+  mockUserFeaturesSpecialty,
+  mockUserFeaturesUnauthenticated
+} from "../../../mock-data/trainee-profile";
+
+const baseCards = ["Support", "MFA"];
 
-const homeCards = [
+const authenticatedCards = [
+  ...baseCards,
   "Profile",
   "Placements",
-  "Programmes",
+  "Programmes"
+];
+
+const specialtyCards = [
+  ...authenticatedCards,
   "Form R (Part A)",
   "Form R (Part B)",
-  "Support",
-  "MFA",
   "Action Summary",
-  "CCT (Certificate of Completion of Training)",
-  "Changing hours (LTFT)"
+  "CCT (Certificate of Completion of Training)"
 ];
 
+const ltftPilotCards = [...specialtyCards, "Changing hours (LTFT)"];
+
 describe("Home with MFA set up", () => {
-  beforeEach(() => {
-    store.dispatch(updatedPreferredMfa("SMS"));
-    store.dispatch(updatedUserFeatures(mockUserFeatures1));
-    mount(
-      <Provider store={store}>
-        <Router history={history}>
-          <Home />
-        </Router>
-      </Provider>
-    );
+  describe("Unauthenticated user", () => {
+    beforeEach(() => {
+      store.dispatch(updatedPreferredMfa("SMS"));
```

Comment:
I think the current implementation for an unauthenticated user ("NOMFA") won't allow them to navigate to the Home page (they just get the Support and MFA set-up options in the nav), so not sure we need to test for unauthenticated user here - unless we want to change the logic?

Reply:
Agree @john12321, it is a bit odd. I had to fudge the test setup a bit by setting the MFA, as you noted.
I could change the "describe" line, as there is a scenario where the backend will disable all features (no profile found). Though I _think_ in the real world that scenario causes the app to not load too 🤷🏻 

Essentially I'm just trying to test the default, no features enabled, state but the app doesn't let you actually get in to that scenario organically.

Reply:
Yeah @Judge40 , the `useCriticalDataLoader` will return an error if the Profile data not loaded so you're right, users never get to that 'default' scenario - unless there are any other real world situations that could cause no features enabled? If not then this is one less thing to worry about and would simplify the logic/tests somewhat.

Reply:
I agree, the current logic means we can delete those tests so if you're happy then I'm happy 😁.
(Sorry AFK for about an hour but will have a look at any changes/ approve when I get back if that's OK?)